vnext/Microsoft.ReactNative/TurboModulesProvider.cpp

```diff
@@ -173,8 +187,39 @@ class TurboModuleImpl : public facebook::react::TurboModule {
                       });
                 }
                 case MethodReturnType::Callback:
-                case MethodReturnType::TwoCallbacks:
-                  VerifyElseCrash(false);
+                case MethodReturnType::TwoCallbacks: {
```

Comment:
>TwoCallbacks [](start = 39, length = 12)

In the C# logic there were some tests that allowed for resolve and reject to be mixed in order on twocallbacks. I never saw it excersized so didn't add it.

Just sharing in case you noticed something similar. If not, we should probably remove that test case from the sample app.

Reply:
@dannyvv, there are a lot of modules that do it one or another way. It is important that we support both cases: resolve first, then reject, or vice versa.

---
In reply to: [440967901](https://github.com/microsoft/react-native-windows/pull/5221#discussion_r440967901) [](ancestors = 440967901)